### ISSUE TITLE
Why does the DOCs GCN explanation differ to the GCN source code?

### ISSUE THREAD

**Issue by EddyDavies** [pyg-team/pytorch_geometric#620]
Why is the [GCN Docs](https://pytorch-geometric.readthedocs.io/en/latest/notes/create_gnn.html) code, which I think I understand, different from the [GCN module](https://github.com/rusty1s/pytorch_geometric/blob/master/torch_geometric/nn/conv/gcn_conv.py) source code?

**Comment by rusty1s** [comment]
The `GCNConv` doc code is a correct, but simplified implementation of the GCN operator. For example, the `GCNConv` module in PyG implements
1. linear transformations via own `Parameter` and `matmul` instead of `torch.nn.Linear`
2. It comes with caching behavior of the normalization
3. It implements the "improved" version from the Graph-UNet paper
4. Optional bias
